inductiva/tasks/methods.py

```diff
@@ -143,3 +143,31 @@ def get(
     ]
 
     return tasks
+
+
+def get_all(
+    status: Optional[Union[str, models.TaskStatusCode]] = None
+) -> List["inductiva.tasks.Task"]:
+    """Get all tasks of a user.
+
+    This function fetches all tasks of a user, sorted by submission
+    time with the most recent first. If status is specified, only
+    tasks with that status will be fetched.
+    Args:
+        status: The status of the tasks to get. If None, tasks with any status
+            will be returned.
+    Returns:
+        List of Task objects.
+    """
+    status = models.TaskStatusCode(status) if status is not None else None
+
+    all_tasks = []
+    page_counter = 1
+
+    while tasks_fetched := _fetch_tasks_from_api(status,
```

Comment:
The `:=` is only supported from python3.8 forward. Is this in agreement with the versions we claim to support?

Reply:
Yes. I think we support from 3.9 forward.

Reply:
yep 3.8+